utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -351,6 +353,38 @@ class WritePreparedTransactionTestBase : public TransactionTestBase {
       : TransactionTestBase(use_stackable_db, two_write_queue, write_policy){};
 
  protected:
+  void DestroyAndReopenWithExtraOptions(size_t snapshot_cache_bits,
```

Comment:
Can you add a TODO(mayabndeh) to provide proper interface to initialize WritePreparedTxnDB without duplicate the ::Open method here?